### PR TITLE
moving server options to references

### PR DESCRIPTION
**Pull request by flossypurse**

### CODE REVIEW

File: docs/references/server-options.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+title: Temporal Server options
+description: undefined
+sidebar_label: Start options
+---
+
+<!-- This file is generated. Do not edit it directly. -->
+
+You can run the Temporal server as a Go application by including the server package `go.temporal.io/server/temporal` and using it to create and start a new Temporal Server.
+
+Note, there are many ways to [run a Temporal Cluster](/kb/install-temporal-cluster-for-development) and this is approach is recommended for a limited number of situations.
```
Comment (aarohib) [inline docs/references/server-options.md:12]:
```suggestion
Note, there are many ways to [run a Temporal Cluster](/kb/install-temporal-cluster-for-development) and this approach is recommended for a limited number of situations.
```

File: docs-src/references/server-options.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
 ---
-id: options
+id: server-options
 title: Temporal Server options
 sidebar_label: Start options
 ---
 
-## Overview
+You can run the Temporal server as a Go application by including the server package `go.temporal.io/server/temporal` and using it to create and start a new Temporal Server.
```
Comment (djmagee) [inline docs-src/references/server-options.md:7]:
```suggestion
You can run the Temporal server as a Go application by including the server package `go.temporal.io/server/temporal` and using it to create and start a Temporal Server.
```
"create...a ~~new~~" per https://developers.google.com/style/word-list#create-new

File: docs-src/references/server-options.md [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
 
-You can run the Temporal server as a Go application by including the server package `go.temporal.io/server/temporal` and using it to create and start a new server:
+Note, there are many ways to [run a Temporal Cluster](/kb/install-temporal-cluster-for-development) and this is approach is recommended for a limited number of situations.
```
Comment (djmagee) [inline docs-src/references/server-options.md:9]:
```suggestion
You can [run a Temporal Cluster](/kb/install-temporal-cluster-for-development) in various ways. We recommend this approach for a limited number of situations.
```

File: docs/references/server-options.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+s := temporal.NewServer(
+	temporal.ForServices(temporal.Services),
+)
+```
+
+### InterruptOn
+
+This option provides a channel that interrupts the server on the signal from that channel.
+
+- If `temporal.InterruptOn()` is not passed, `server.Start()` is not blocking and `server.Stop()` needs to be called somewhere.
```
Comment (aarohib) [inline docs/references/server-options.md:61]:
```suggestion
- If `temporal.InterruptOn()` is not passed, `server.Start()` is never blocked and you need to call `server.Stop()` somewhere.
```

**Review by djmagee — APPROVED** [review]

File: docs/references/server-options.md [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+- If `temporal.InterruptOn(nil)` is passed, `server.Start()` blocks forever until the process is killed.
+- If `temporal.InterruptOn(temporal.InterruptCh())` is passed, `server.Start()` blocks until Ctrl+C is used and then the server is gracefully shut down.
```
Comment (aarohib) [inline docs/references/server-options.md:63]:
```suggestion
- If `temporal.InterruptOn(temporal.InterruptCh())` is passed, `server.Start()` blocks until you use Ctrl+C, which then gracefully shuts the server down.
```

**Pull request merged**